File: app/src/main/java/com/example/schedule/POJO/OK_POJO/Response.java
package com.example.schedule.POJO.OK_POJO;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Response {

    @SerializedName("response")
    @Expose
    private Obj obj;

    public Obj getObj() {
        return obj;
    }

    public void setObj(Obj obj) {
        this.obj = obj;
    }
}

File: app/src/main/java/com/example/schedule/POJO/OK_POJO/Response.kt
package com.example.schedule.POJO.OK_POJO

import com.example.schedule.POJO.ERROR_POJO.Error
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Response {
    @SerializedName("response")
    @Expose
    var jsonResponse: JsonResponse? = null

    @SerializedName("error")
    @Expose
    var error: Error? = null

}
